LibraBFT/Impl/Consensus/RoundManager/Properties.agda

```diff
@@ -74,14 +73,29 @@ module ProcessProposalMSpec (proposal : Block) where
   open import LibraBFT.Impl.Consensus.Liveness.Properties.ProposerElection
   open import LibraBFT.Impl.Consensus.BlockStorage.Properties.BlockStore
 
+  -- TODO-2: this needs to change because processProposalM can have logging outputs; an attempt below
   OutputSpec : ErrLog ⊎ Unit → List Output → Set
   OutputSpec (inj₁ _) outs = outs ≡ []
   OutputSpec (inj₂ _) outs = ∃₂ λ mv pid → outs ≡ SendVote mv pid ∷ []
 
+  -- TODO-1: this should be near the definition of Output
+  isSendVote : Output → Set
+  isSendVote out = ∃₂ λ mv pid → out ≡ SendVote mv pid
+
+  isSendVote? : (out : Output) → Dec(isSendVote out)
```

Comment:
Only a very minor point, but I prefer capitalization for definitions of types (I can go ahead and make this change when I merge this PR in).